actionlist/app/reducers/action_items.js

```diff
@@ -1,12 +1,23 @@
 /* eslint arrow-body-style: ["error", "as-needed", { "requireReturnForObjectLiteral": true }] */
 import { handleActions } from 'redux-actions';
-import { TOGGLE_DRAWER, SORT_ACTION_LIST } from '../actions/types';
+import { TOGGLE_DRAWER, SORT_ACTION_LIST, SELECT_DROPDOWN_OPTION } from '../actions/types';
 import { processInstances, sortTypes } from '../lib/commons';
 
 export const defaultState = {
   dataSource: processInstances,
   drawerExpanded: false,
   optionSelected: '',
+  dropdownColors: {
+    saved: 'white',
+    initiated: 'white',
+    disapproved: 'white',
+    enroute: 'white',
+    approved: 'white',
+    final: 'white',
+    processed: 'white',
+    exception: 'white',
+    cancel: 'white',
+  },
```

Comment:
This is to change the dropdown background color on selection. By default it is all white.